packaging/package/package.py

```diff
@@ -18,7 +19,7 @@ def create_package_output(path):
 
 
 def main():
-    path_from_args = Path("D:/dev/tableau/connector-plugin-sdk/samples/plugins/postgres_odbc")
+    path_from_args = Path("..\..\samples\plugins\postgres_odbc")
```

Comment:
Good idea to point all these to `postgres_odbc` 😄 I added `TODO`s to them so they're easy to find.

```suggestion
    path_from_args = Path("..\..\samples\plugins\postgres_odbc"). # TODO: replace when ready
```